### DISCUSSION TITLE
[For the community] Demo OS based on a light Linux distro like Alpine? Similar to wmlive or to live GNUstep

### DISCUSSION THREAD

**Discussion by xplshn** [trunkmaster/nextspace#433] [Ideas]
Constructing an OS based on Alpine would only require creating an APKBUILD template to successfully build NeXTSPACE and then produce an Image/ISO with it installed. It'd be great because Alpine is light and fast. It could also be used as a demo, much like https://distrowatch.com/table.php?distribution=gnustep and https://wmlive.sourceforge.net

Using Alpine as a base would be great because:
 - MUSL does not break across versions as much as GLibc
 - MUSL is POSIX and ISO C compliant
 - Software compiled against lightweight dependencies is usually lighter & faster
 - Can be optionally built with LLVM & Clang though it requires some work
 - Complete repos. (community, main and edge/testing). (-static libraries are available for most packages in case they are needed, which is great given that most distros don't ship these anymore in their -dev packages)
 - Reproducible builds!
 - Alpine can be built in less time than Fedora, Debian, Arch. As it doesn't ship with heavy bloatware such as GLibc, Systemd, or udev by default

**Comment by lproven** [comment]
I like this idea very much and 100% endorse it.
